### ISSUE TITLE
MariaDB connector does not support BIT datatype

### ISSUE THREAD

**Issue by fwy** [trinodb/trino#16891]
Trying the MariaDB connector on a test database, it appears that fields with the datatype of BIT are not supported and skipped by Trino.

I think that the following code could be added to the MariaDbClient.toColumnMapping() method to fix this, although I did not test it.

            case Types.BIT:
                return Optional.of(booleanColumnMapping());

At least, this code works in a custom connector I wrote to address the same issue in a different database.

Does this make sense, and if it works should I create a pull request for it?

**Comment by ebyhr** [comment]
I don't think we want to map MariaDB BIT type to Trino boolean because BIT type can accept several bit values (= not limited to true or false).

**Comment by fwy** [comment]
OK.  How about mapping it to a string?  It may not be ideal, but I think it is better than simply skipping a field in the data source that may have important contents.

**Comment by ebyhr** [comment]
I suppose we can already map to string with `jdbc-types-mapped-to-varchar` config property.

**Comment by ebyhr** [comment]
No need to close this issue. Someone may want to implement the support.
